ngrok/src/tunnel_ext.rs

```diff
@@ -1,16 +1,21 @@
 #[cfg(target_os = "windows")]
 use std::time::Duration;
+use std::{
+    borrow::Cow,
```

Comment:
Looks like borrow::Cow isn't used on windows, so an unused import warn building on the windows box.
Could pull to a separate line and add the `#[cfg(not(target_os = "windows"))]`